draft-ietf-rats-architecture.md

```diff
@@ -1278,7 +1281,65 @@ valid.  Thus, if the Attestation Result sends a validity lifetime
 in terms of time(RX)-time(RG), then the Relying Party can check
 `time(OP) - time(NS') < time(RX)-time(RG)`.
 
-## Example 3: Timestamp-based Background-Check Model Example
+## Example 3: Handle-based Passport Model Example
+
+Handles are a third option to establish time-keeping next to nonces or timestamps.
+Handles are opaque data intended to be available to all RATS roles that interact with each other, such as the Attester or Verifier, in specified intervals.
+To enable this availability, handles are distributed centrally by the Handle Distributor role over the network.
+As any other role, the Handle Distributor role can be taken on by a dedicated entity or collapsed with other roles, such as a Verifier.
+The use of handles can compensate for a lack of clocks or other sources of time on entities taking on RATS roles.
+The only entity that requires access to a source of time is the entity taking on the role of Handle Distributor.
+
+Handles are different to nonces as they are not used only once and are used by more than one entity at the same time.
+Handles are different to timestamps as they do not have to convey information about a point in time, but their reception creates that information.
+The reception of a handle is similar to the event that increments a relative tickcounter.
+The event at which a handle is received a new (most recent) handle outdates (and therefore invalidates) the former received handle.
+
+In this example, Evidence generation based on received handles always uses the current (most recent) handle.
+As handles are distributed over the network, all involved entities receive a fresh handle at roughly the same time.
+The time that represents this event is named Handle received: time(HR). Due to distribution over the network, there is some jitter with respect to time(HR) for each involved entity.
+To compensate for this jitter, there is a small period of overlap (a specified offset) in which both a current handle and corresponding former handle are valid in Evidence appraisal: `validity-duration = time(HR') + offset - time(HR)`. The offset is typically based on a network's round trip time.
+Analogously, the generation of valid Evidence is only possible, if the age of the handle used is lower than the validity-duration: `delta(time(EG),time(HR) < validity-duration`.
```

Comment:
Mismatched parens.  Also other sections use subtraction rather than delta().  We should pick one style and be consistent.